include/seqan/platform/platform_windows.h

```diff
@@ -71,6 +71,11 @@
 #pragma warning( disable : 4345 )
 #pragma warning( disable : 4351 )
 
+// allow multiple constructors
+#pragma warning( disable : 4521 )
```

Comment:
Oh the warnings happen on develop but not on master... @rrahn @h-2 any ideas why?